set-max-extent/README.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "category": "Maps and Scenes",
+    "description": "Limit the view of a map to a particular area.",
+    "formal_name": "SetMaxExtent",
+    "ignore": false,
+    "images": [
+        "set-max-extent.png"
+    ],
+    "keywords": [
+        "extent",
+        "limit panning",
+        "map",
+        "mapview",
+        "max extent",
```

Comment:
I think "max" on its own would be a good keyword, but would that be included with this one?

Reply:
Yes, it should!